README.md

```diff
@@ -212,18 +212,41 @@ Said differently, if `H` directly succeeds `G`, then `H` MUST be preferred over
 `G`.
 
 The implications of determining the "preferred epochs" will be addressed in
-section 4.7.
+section 4.8.
 
 
-### 4.3. Resolving forked epochs with same membership
+### 4.3. Tie-breaking rule
 
-Suppose there are two forked epochs `L` and `R`, and `L`'s epoch key
-encoded in hexadecimal is lexicographically less than `R`'s epoch key encoded in
-hexadecimal.
+In some cases, there may be multiple forked epochs that "tie" as the preferred
+epoch.  We define a tie-breaking rule that allows all remaining members to
+deterministically choose one of those tied epochs as the preferred epoch.  This
+rule will be used in sections 4.4. and 4.6.
+
+Suppose:
+
+- `keys` is the set of all epoch keys that are currently "tied"
+- `keys_count` is the size of the set `keys`
+- `sorted_keys` is the lexicographically sorted (by hexadecimal encoding) list
+  of all epoch keys in `keys`
+- `concat_sorted_keys` is the concatenation of all epoch keys in `sorted_keys`
+- `hashed` is the SHA-256 hash of `concat_sorted_keys`
+- `hashed8` are the first 8 bytes of `hashed`
+- `as_number` is the integer value of `hashed8` in big-endian byte order
```

Comment:
I can never remember which endian is which but I guess you checked

Reply:
Do we maybe want to provide a minimal example people can test their rule implementations against?

Reply:
Big-endian is basically the way we write numbers on paper: 123 i.e. one hundred, the "1" there means the biggest digit meaning one hundred, while the "3" is the smallest.

Little-endian is weird, it would be "321" for "one hundred and twenty three" but LE is very useful in the low level of computers.

Reply:
> Do we maybe want to provide a minimal example people can test their rule implementations against?

Yes! As #14

Reply:
But doesn't that mean that little endian is usually the way number parsers work?

Reply:
It highly depends. Like `parseInt` in JS is big-endian.